src/main/resources/templates/navbar.html

```diff
@@ -1,35 +1,30 @@
 <html xmlns:th="http://www.thymeleaf.org"
```

Comment:
same comment as in the footer

Reply:
in my opinion the tags are needed here

Reply:
did you try and look at the page with f12? is there multiple html and body tags?

Reply:
![Bildschirmfoto 2024-07-22 um 14 05 15](https://github.com/user-attachments/assets/2fd282dd-6c6e-49fa-8f0c-3b76a75958d7)
there are no multiple HTML Tags

Reply:
alright, then it should be fine :)